shared/AppInsightsCore/src/JavaScriptSDK/CoreUtils.ts

```diff
@@ -396,6 +396,26 @@ export class CoreUtils {
         return false;
     }
 
+    /**
+     * Try to determine if the object is extensible when run within an >= ES5 container.
+     * For < ES5 containers where the method does not exist, always return true.
+     * @param target The object on which to determine extensibility.
+     * @returns True if the object is extensible.
+     */
+    public static isExtensible(obj:object) : boolean {
+        let extensibleProp = Object["isExtensible"];
+        if (extensibleProp) {
+            try {
+                return extensibleProp(obj);
```

Comment:
From a code minification perspective, this could be made *slightly* better by having 1 return statement